src/components/Shared/Types/Matter.tsx

```diff
@@ -0,0 +1,6 @@
+export type Matter = {
+  /** the name of the matter, usually a legislation name */
+  name: string;
+  /** the id of the matter */
+  id: string;
+};
```

Comment:
If this is usually referring to a piece of legislation could we potentially rename all instances of "Matter" to "Legislation" or "LegislationName" so that the code is a bit more descriptive to the reader? Or if we still need to be flexible maybe something like "MeetingSubject"?

I ask because even though I had a sense of what "Matter" probably meant in this context, I still had to search around for this file/code comments to find exactly what "Matter" meant in other files.

Reply:
This is the general form of the idea of a code change! Matters refer to legislation, initiatives, and resolutions! It also mirrors the Legistar API. Not ideal but can always check the schema docs: https://councildataproject.github.io/cdp-backend/cdp_backend.database.html#cdp_backend.database.models.Matter

Reply:
Ooh okay got it! If that's the case, then we can totally leave it as is!